CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 4.7.0 (February 9, 2024)
+
+### NEW - RESOURCES, DATA SOURCES, PROPERTIES, ATTRIBUTES, ENV VARS:
+
+* New datasource: `okta_default_signin_page` retrieve the default signin page of a brand Okta [#1842](https://github.com/okta/terraform-provider-okta/pull/1842). Thanks, [@duytiennguyen-okta](https://github.com/duytiennguyen-okta)!
+* New datasource: `okta_log_stream` retrieve log stream [#1843](https://github.com/okta/terraform-provider-okta/pull/1843). Thanks, [@monde](https://github.com/monde)!
```

Comment:
okta log stream is randomVariable2's work in #1830, I just brought it in to use the correct sdk client.